kong/plugins/ai-proxy/schema.lua

```diff
@@ -0,0 +1,146 @@
+local typedefs = require "kong.db.schema.typedefs"
+
+local request_log_compatible = {
+  "llm/v1/chat",
+  "llm/v1/completions",
+}
+
+local auth_schema = {
+  type = "record",
+  required = false,
+  fields = {
+    { header_name = { type = "string", description = "If AI model requires authentication via Authorization or API key header, specify its name here.", required = false, referenceable = true } },
```

Comment:
why are header name and param name referenceable?

Reply:
Just from past experience what users go asking for!